docs/helpers/for-of.md

```diff
@@ -65,6 +65,32 @@
   ```
   @codepen
 
+  If a positive integer value is passed, it will loop that number of times.
```

Comment:
We should probably clean up the signature docs, specifically around this part:

```
EXPRESSION {KeyLookup Expression|Call Expression}: An expression that typically returns a list like data structure. If the value of the EXPRESSION is an observable list (for example: can-define/list/list), the resulting HTML is updated when the list changes. When a change in the list happens, only the minimum amount of DOM element changes occur. The list itself can also change, and a difference will be performed, which also will perform a minimal set of updates.
```

I think we should make it more clear what happens if the `expression` evaluates to:

- An array-like structure (linking to canReflect.isListLike would be good here)
- An object-like (key-value) data structure
- A Number